Assets/MixedRealityToolkit/_Core/Definitions/Devices/SupportedControllerType.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices
+{
+    /// <summary>
+    /// The SDKType lists the XR SDKs that are supported by the Mixed Reality Toolkit.
+    /// Initially, this lists proposed SDKs, not all may be implemented at this time (please see ReleaseNotes for more details)
+    /// </summary>
+    public enum SupportedControllerType
```

Comment:
Shouldn't we have Hand as a supported controller?

Reply:
Not if you're referring to the WMR hand input. Then it's just a Windows Mixed Reality Controller type.